src/librustc_resolve/lib.rs

```diff
@@ -1570,6 +1573,7 @@ impl<'a> Resolver<'a> {
                                 TypeNS => Scope::ExternPrelude,
                                 ValueNS => Scope::StdLibPrelude,
                                 MacroNS => Scope::RegisteredAttrs,
+                                LifetimeNS => bug!("shouldn't be resolving things in lifetime ns"),
```

Comment:
Judging by the other code below: `LifetimeNS => break, // nowhere else to search`